apps/search-poc/src/index.tsx

```diff
@@ -2,18 +2,21 @@ import * as React from 'react';
 import * as serviceWorker from './serviceWorker';
 import { render } from 'react-dom';
 import { Switch, Route } from 'react-router';
-import { BrowserRouter as Router } from 'react-router-dom';
```

Comment:
I guess technically we don't need the qRouter` anymore and can go back to the BrowserRouter, which will save us having to created the `src/history.ts` file

Reply:
I've fixed that as well.